liberty-maven-plugin/src/it/setup/test-war/pom.xml

```diff
@@ -128,6 +128,7 @@
                                     <version>${runtimeVersion}</version>
                                     <type>zip</type>
                                 </assemblyArtifact>
+                                <feature>osgiBundle-1.0</feature>
```

Comment:
Why was this added?

Reply:
Forgot to remove this when I was finished testing. Removing 👍